### PR TITLE
clad and cmssw proposals

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

File: _gsocproposals/2021/proposal_functor_clad.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+     int main () {
+       Functor Fn;
+       Fn.x = 2;
+       double pow2 = Fn();
+       auto dpow2_dx = clad::differentiate(Fn, /*wrt*/ 0); // unsupported
+       return pow2;
+     }
+
+The goal of this project is to modify Clad to handle suhc cases.
```
Comment (aflp91) [inline _gsocproposals/2021/proposal_functor_clad.md:28]: suhc => such

File: _gsocproposals/2021/proposal_secondorder_clad.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+ROOT is a framework for data processing, born at CERN, at the heart of the research on high-energy physics. Every day, thousands of physicists use ROOT applications to analyze their data or to perform simulations. ROOT has a clang-based C++ interpreter Cling and integrates with Clad to enable flexible automatic differentiation facility.
+
+## Task ideas and expected results
+
+TFormula is a ROOT class which bridges compiled and interpreted code. Teach TFormula to use second order derivatives (using clad::hessian). The implementation should very similar to what was done in [this pull request](https://github.com/root-project/root/pull/2745). The produced code should be well tested and documented. The student should be prepared to write a progress report and present the results. If time permits, we should pursue converting the C++ gradient function to CUDA device kernels. The integration of that feature should be comparable in terms of complexity to integrating 'clad::hessians'.
```
Comment (aflp91) [inline _gsocproposals/2021/proposal_secondorder_clad.md:17]: should very similar => should be very similar ?

**Review by aflp91 — CHANGES_REQUESTED** [review]
Two minor typos; otherwise looks OK with me.

**Review by xvallspl — APPROVED** [review]

**Comment by xvallspl** [comment]
see the changes suggested by Antoine

**Review by agheata — APPROVED** [review]

**Review by aflp91 — APPROVED** [review]
OK with me!

**Pull request merged**